### PR TITLE
BAC-50 Creating a utility function to upload files

### PR DESCRIPTION
**Pull request by rector619**
https://linear.app/team-socket/issue/BAC-50/creating-a-utility-function-to-upload-files

### CODE REVIEW

**Review by mikengr — COMMENTED** [review]
I honestly don't understand this PR. The endpoint implementation and ticket title seems quite divergent.

**Comment by rector619** [comment]
- Created an utility function to upload files using DB upload
- Handled error cases by raising direct http exception
- Wrote tests for the utility function covering all the cases

File: backend/endpoints/rooms.py [hunk 1]
```diff
@@ -439,6 +439,8 @@ async def get_members(org_id: str, room_id: str):
         ),
     )
 
+<<<<<<< HEAD
```
Comment (zxenonx) [inline backend/endpoints/rooms.py]: please remove these merge conflicts markers

File: backend/endpoints/rooms.py [hunk 2]
```diff
@@ -439,6 +439,8 @@ async def get_members(org_id: str, room_id: str):
...
+=======
```
Comment (zxenonx) [inline backend/endpoints/rooms.py]: please remove these merge conflicts markers

File: backend/endpoints/rooms.py [hunk 3]
```diff
@@ -616,4 +618,5 @@ async def delete_room(org_id: str, room_id: str):
             data=room,
             message="Room Deleted successfully",
         ),
-    )    
\ No newline at end of file
+    )    
+>>>>>>> cb63f47124335054616951996835992142a5dc15
```
Comment (zxenonx) [inline backend/endpoints/rooms.py]: please remove these merge conflicts markers

**Review by zxenonx — CHANGES_REQUESTED** [review]
Good work.
@RECTOR619 
I left some comments

File: backend/endpoints/rooms.py (hunk 1 above)
Comment (rector619) [reply backend/endpoints/rooms.py]: i will now sir

File: backend/endpoints/rooms.py (hunk 3 above)
Comment (rector619) [reply backend/endpoints/rooms.py]: i will now sir

File: backend/endpoints/rooms.py (hunk 2 above)
Comment (rector619) [reply backend/endpoints/rooms.py]: i will now sir

Comment (rector619) [reply backend/endpoints/rooms.py]: > please remove these merge conflicts markers

have resolved the conflict sir

**Comment by mikengr** [comment]
Duplicated task

**Pull request closed without merging**